dev-utils/src/commands/init_command.rs

```diff
@@ -307,10 +310,35 @@ impl InitCommand {
         assert!(address.to_string().trim().to_string().len() == 42);
         Ok(())
     }
+
+    fn clear_disk(&self) -> Result<(), Error> {
+        let _ = fs::remove_dir_all(format!(
+            "{}/.tendermint/config",
+            dirs::home_dir().unwrap().to_str().unwrap()
+        ));
+        let _ = fs::remove_dir_all(format!(
+            "{}/.tendermint/data",
+            dirs::home_dir().unwrap().to_str().unwrap()
+        ));
+
+        let _ = fs::canonicalize(PathBuf::from("./.cro-storage")).and_then(|p| {
+            let _ = fs::remove_dir_all(p);
+            Ok(())
+        });
+
+        let _ = fs::canonicalize(PathBuf::from("./.storage")).and_then(|p| {
+            let _ = fs::remove_dir_all(p);
+            Ok(())
+        });
+
+        Ok(())
+    }
+
     pub fn execute(&mut self) -> Result<(), Error> {
-        println!("initialize");
+        println!("initialize chain");
 
-        self.prepare_tendermint()
+        self.clear_disk()
```

Comment:
perhaps display some warning or ask for confirmation that it'll delete the stuff

Reply:
ok, i'll add warning and confirmation

Reply:
i added warning, double check for clearing folders